Sources/App/Views/PackageController/Builds/BuildShow+View.swift

```diff
@@ -75,20 +75,25 @@ private extension Build.Status {
         switch self {
             case .ok: return "Build Succeeded"
             case .failed: return "Build Failed"
+            case .pending: return "Build Pending"
         }
     }
 
     var cssClass: String {
         switch self {
             case .ok: return "green"
             case .failed: return "red"
+            // FIXME: what class to use?
+            case .pending: return "grey"
```

Comment:
We have a choice of creating the build stubs with `status = NULL` or `status = 'pending'` but either way we'd need to handle the extra state here. Is there an existing class I could/should use?

Reply:
I've added this class for now, but do we really want to let people get to this page for a pending build? I guess they can see the build command that's going to be executed, but it might be worth making it the same as the "unknown" state where this page isn't accessible.

Either way, I've added the CSS for this so it'll work if we decide to keep this use case in. Let's chat about it in the morning.